src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -1,13 +1,46 @@
 package org.teamresistance.frc;
 
 import edu.wpi.first.wpilibj.IterativeRobot;
+import org.strongback.Strongback;
 
 /**
  * Main robot class. Override methods from {@link IterativeRobot} to define behavior.
+ *
+ * @author Shreya Ravi
  */
 public class Robot extends IterativeRobot {
 
-    @Override
-    public void robotInit() {
-    }
+  private final Drive drive = new Drive(IO.robotDrive, IO.leftJoystick.getRoll(), IO.leftJoystick.getPitch(), IO.rightJoystick.getRoll());
```

Comment:
While we don't have an official line length maximum, this line feels too long to be easy to read. How do you feel about breaking it like this:

```java
  private final Drive drive = new Drive(IO.robotDrive, IO.leftJoystick.getRoll(), 
      IO.leftJoystick.getPitch(), IO.rightJoystick.getRoll());
```